modules/ROOT/pages/sgcollect-info.adoc

```diff
@@ -131,6 +131,18 @@ The tool creates the following log files in the ouput file.
 
 |1.3.1
 |1.6.3
+
+|1.4
+|TODO
+
+|1.5
+|TODO
+
+|2.0
+|TODO
+
+|2.1
+|TODO
```

Comment:
@adamcfraser/@bbrks ^^^ this table needs info.

Reply:
this table is rendered here https://docs.couchbase.com/sync-gateway/2.1/sgcollect-info.html#installing-optional-dependencies (currently missing Go versions for SG 1.4 and above)